velox/dwio/dwrf/writer/StatisticsBuilder.cpp

```diff
@@ -115,8 +122,12 @@ std::unique_ptr<dwio::common::ColumnStatistics> StatisticsBuilder::build()
   proto::ColumnStatistics stats;
   toProto(stats);
   StatsContext context{WriterVersion_CURRENT};
-  return buildColumnStatisticsFromProto(
-      ColumnStatisticsWrapper(&stats), context);
+  auto result =
+      buildColumnStatisticsFromProto(ColumnStatisticsWrapper(&stats), context);
+  if (hll_) {
+    result->setNumDistinct(hll_->cardinality());
```

Comment:
Can we take care of this in buildColumnStatisticsFromProto or proto::ColumnStatistics doesn't have num distinct? Thanks!